.github/workflows/release.yaml

```diff
@@ -498,6 +498,7 @@ jobs:
 
       - name: SBOM Generation and Attestation
         if: ${{ !inputs.dry_run }}
+        continue-on-error: true
```

Comment:
I think for the release workflow, we should fail. WDYT?

Reply:
Sure, that makes sense.